<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="journeys"
    errors="1" failures="0" hostname="" id="" package="journeys" skipped="0"
    tests="1" time="0.003000" timestamp="Sun, 29 Apr 2018 17:46:45 GMT">
  
    <testcase name="takes user to the item page" classname="journeys" time="0.003000" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [Journeys / takes user to the item page]:

  ReferenceError: port is not defined
    at Object.takes user to the item page (/Users/yerbue01/workspace/vue-hackernews/e2e/specs/journeys.js:4:32)
    at Module.call (/Users/yerbue01/workspace/vue-hackernews/node_modules/nightwatch/lib/runner/module.js:62:34)
    at /Users/yerbue01/workspace/vue-hackernews/node_modules/nightwatch/lib/runner/testcase.js:70:29
    at _fulfilled (/Users/yerbue01/workspace/vue-hackernews/node_modules/nightwatch/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/yerbue01/workspace/vue-hackernews/node_modules/nightwatch/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/yerbue01/workspace/vue-hackernews/node_modules/nightwatch/node_modules/q/q.js:796:13)
    at /Users/yerbue01/workspace/vue-hackernews/node_modules/nightwatch/node_modules/q/q.js:556:49
    at runSingle (/Users/yerbue01/workspace/vue-hackernews/node_modules/nightwatch/node_modules/q/q.js:137:13)
    at flush (/Users/yerbue01/workspace/vue-hackernews/node_modules/nightwatch/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    </system-err>
  

  
  </testsuite>
</testsuites>
